docs/source/spark.rst

```diff
@@ -123,6 +122,27 @@ Reasons you might choose Dask
    multiple packages
 
 
+Reaons to choose both
+---------------------
+
+It is easy to use both Dask and Spark on the same data and on the same cluster.
+
+They can both read and write common formats, like CSV, JSON, ORC, and Parquet,
+making it easy to hand results off between Dask and Spark workflows.
+
+They can both deploy on the same clusters.
+Most clusters are designed to support many different distributed systems at the
+same time, using resource managers like Kubernetes and YARN.  If you already
+have a cluster on which you run Spark workloads, it's likely easy to also run
+Dask workloads on your current infrastructure and vice versa.
+
+In particular, for users coming from traditional Hadoop/Spark clusters (such as
+those sold by Cloudera/Hortonworks) you are likely using the Yarn resource
```

Comment:
Not likely, this is always true. I'd just delete this sentence and merge this paragraph with the above.

Reply:
I think that it might be useful to call out Cloudera/Hortonworks for the readers that identify that they have such a cluster, but don't understand what Yarn is.

Reply:
My disagreement is with the "likely using the Yarn resource manager" part - this is always true for a Hadoop cluster. Perhaps just drop that part then:

```
For users coming from traditional Hadoop/Spark clusters (such as those sold by Cloudera/Hortonworks), you can deploy dask on these systems...
```